frontend/packages/ux-editor/src/components/config/editModal/EditOptions.tsx

```diff
@@ -57,6 +57,7 @@ export function EditOptions<T extends SelectionComponentType>({
   const initialSelectedOptionType = getSelectedOptionsType(component.optionsId, component.options);
   const [selectedOptionsType, setSelectedOptionsType] = useState(initialSelectedOptionType);
   const { t } = useTranslation();
+  const [optionIds, setOptionIds] = useState<string[]>([]);
```

Comment:
Kan det være en ide endre denne til å være en variabel? Da slipper vi useEffect? 😊 
```suggestion
  const mappedOptionIds = useMemo(() => component.options.map((_, index) => `option_${index}`, [components.options]);
```

Reply:
Endret😊